tools/debian_buildenv.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Can you add a comment here that this script is also supposed to work with Ubuntu et. al.?